public/docs/ts/latest/guide/browser-support.jade

```diff
@@ -0,0 +1,190 @@
+:marked
+  # Browser support #  
+  Angular supports most recent browsers. This includes the following specific versions:
+
+table
+  tr
+    th Chrome
+    th Firefox
+    th Edge
+    th IE
+    th Safari
+    th iOS
+    th Android
+    th IE mobile
+  tr
+    td latest
+    td latest
+    td 14
+    td 11
+    td 9
+    td 9
+    td Marshmallow (6.0)
+    td 11
+  tr
+    td
+    td
+    td 13
+    td 10
+    td 8
+    td 8
+    td Lollipop<br>(5.0, 5.1)
+    td
+  tr
+    td
+    td
+    td
+    td 9
+    td 7
+    td 7
+    td KitKat<br>(4.4)
+    td
+  tr
+    td
+    td
+    td
+    td
+    td
+    td
+    td Jelly Bean<br>(4.1, 4.2, 4.3)
+    td
+    
+    
+:marked
+  To ensure this compatibility, the unit tests of the framework are run on all of them for every single commit. 
```

Comment:
"every single commit" sounds a bit dramatic and it's not really true. we run tests on PRs which can contain multiple commits.